xla/service/gpu/gpu_compiler.cc

```diff
@@ -996,6 +996,9 @@ absl::Status RunLayoutAssignmentPasses(HloModule* hlo_module,
   pipeline.AddPass<SubByteNormalization>(
       SubByteNormalization::SET_ELEMENT_SIZE);
   pipeline.AddPass<OptimizeInputOutputBufferAlias>(true);
+  pipeline.AddPass<HostOffloadLegalize>(
```

Comment:
Please add a comment that this needs to run before LayoutNormalization.
The PR description explains it, but for a later reader of the code, they will usually not check in which PR this got added and will lack the context why these passes appear as part of LayoutAssignmentPasses.

Reply:
Done.